orion/sites/admin/src/announce/announce.js

```diff
@@ -2,12 +2,70 @@
 require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
+import { Link } from 'react-router-dom';
 
 export class AnnouncePage extends React.Component {
 	render() {
+		var numAnnouncements = 3; 	/* this needs to be a variable later */
+
+		/* these fake announcements will eventually be replaced with real announcements */
+		let fakeAnnounceA = {
+			date: "02/15/2020",
+			author: "Melon King",
+			message: "Only evil people waste watermelon"
+		};
+		let fakeAnnounceB = {
+			date: "02/16/2020",
+			author: "Fruit Hater",
+			message: "Watermelons are fruit, therefore they deserve to dry on the vine"
+		};
+		/* will use this to test text wrapping later */
+		let fakeAnnounceC = {
+			date: "02/23/2020",
+			author: "Talkative",
+			message: "This will be a longer message later"
+		};
 		return (
       <div id="view-announce">
-        <h1>All Announcements</h1>
+        <h1>All Announcements ({numAnnouncements}) </h1>
+				<ul> {/* this is the subheading */}
+					<div className="announce-lists">
+						<li className="li-med"> Date </li>
+						<li className="li-med"> Author </li>
+						<li className="li-large"> Message </li>
+						<li className="li-small"> </li>
+					</div>
+				</ul>
+
+				{/* these are the lists under the subheading */}
+				<ul> {/*I think there's a problem having li under li, but I'll fix it later*/}
+					<li className="announce-lists">
```

Comment:
Similarly here, `announce-lists` isn't the best name since technically this is not a list, it's an individual item of a list. So I suggest a different name like `announce-list-item` or `announce-list-row`. Whatever name makes sense for you. Still keep it a `className` since there are multiple of these `<li>`'s.